### DISCUSSION TITLE
2008/08/28/%E5%88%A0%E9%99%A4live-space%E4%B8%8A%E5%9E%83%E5%9C%BE%E7%95%99%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#240] [Announcements]
# 2008/08/28/%E5%88%A0%E9%99%A4live-space%E4%B8%8A%E5%9E%83%E5%9C%BE%E7%95%99%E8%A8%80

最近垃圾留言很多，今天google了一下对付垃圾留言的方法，还真找到了一个。

https://blog.qizhen.xyz/2008/08/28/%E5%88%A0%E9%99%A4live-space%E4%B8%8A%E5%9E%83%E5%9C%BE%E7%95%99%E8%A8%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：

dong
2008年09月13日 12:35 编辑
赞

回复
永青
2008年08月28日 22:11 编辑
呵呵，果然有用。

回复
执子之手
2008年08月28日 22:07 编辑
这招有意思~

回复
Ophelia Dongmei
2008年08月28日 22:00 编辑
真是个好办法！

回复
心
2008年08月28日 21:12 编辑
好主意！

回复
David
2008年08月28日 20:58 编辑
恩

回复
